basis/windows/ole32/ole32-tests.factor

```diff
@@ -29,3 +29,7 @@ IN: windows.ole32.tests
 [ "{01234567-89ab-cdef-0123-456789abcdef}" ]
 [ "{01234567-89ab-cdef-0123-456789abcdef}" string>guid guid>string ]
 unit-test
+
+[ f ] [
+    f create-guid 10 [ create-guid [ guid= ] keep -rot or swap ] times drop
```

Comment:
It might be better to use ``any?`` so you don't have to do the stack shuffling:

```factor
{ f } [ create-guid 10 iota [ drop create-guid guid= ] with any? ]
```